lib/node_modules/@stdlib/string/reverse/docs/types/test.ts

```diff
@@ -36,6 +36,33 @@ import reverseString = require( './index' );
 	reverseString( [] ); // $ExpectError
 	reverseString( {} ); // $ExpectError
 	reverseString( ( x: number ): number => x ); // $ExpectError
+
+	reverseString( true, {} ); // $ExpectError
+	reverseString( false, {} ); // $ExpectError
+	reverseString( null, {} ); // $ExpectError
+	reverseString( undefined, {} ); // $ExpectError
+	reverseString( 5, {} ); // $ExpectError
+	reverseString( [], {} ); // $ExpectError
+	reverseString( {}, {} ); // $ExpectError
+	reverseString( ( x: number ): number => x, {} ); // $ExpectError
+}
+
```

Comment:
We should also add tests here when not provided an object for the second argument.